src/app-layout/__tests__/mobile.test.tsx

```diff
@@ -447,8 +447,16 @@ describeEachThemeAppLayout(true, theme => {
 
   test('Does not add a label to the toggle and landmark when they are not defined', () => {
     const { wrapper } = renderComponent(<AppLayout contentType="form" {...drawerWithoutLabels} />);
-    const findDrawersToolbar = () => wrapper.findByClassName(drawerBarClassName);
+    const findMobileToolbar = () => wrapper.findByClassName(mobileBarClassName);
+    const drawersAside = within(findMobileToolbar()!.getElement()).getByRole('region');
     expect(wrapper.findDrawerTriggerById('security')!.getElement()).not.toHaveAttribute('aria-label');
-    expect(findDrawersToolbar()!.getElement()).not.toHaveAttribute('aria-label');
+    expect(drawersAside).not.toHaveAttribute('aria-label');
+  });
+
+  test('renders correct mark-up and roles', () => {
```

Comment:
Can we sync this test and the one above?

So they use the same finding strategy and one checks that the found element has label and role and the other one has only role